Sources/Rugby/Common/LogPrinter/Printers/RugbyPrinter.swift

```diff
@@ -47,19 +53,26 @@ final class RugbyPrinter: Printer {
     @discardableResult
     func spinner<Result>(_ text: String, job: @escaping () throws -> Result) rethrows -> Result {
         defer { print("\(text) \("✓".white)".yellow, level: .vv) }
-        if skipSpinner { return try job() }
-        isSpinnerAnimating = true
-        let result = try Spinner().show(text: text, job)
-        isSpinnerAnimating = false
-        return result
+        switch spinnerMode {
+        case .quiet:
+            return try job()
+        case .simple:
+            print(text)
```

Comment:
I think we should use here non standard Swift print method:
```suggestion
            print(text, level: .vv)
```